generator/src/lib/config.ts

```diff
@@ -1 +1,2 @@
-export const DOCUMENTATION_URL = "/documentation"
+export const DOCUMENTATION_URL: string = "/documentation"
+export const ISS_DOMAIN = ["tags.ioxio.dev"]
```

Comment:
Why is this an array?

Reply:
I misunderstood something about this. Ok its a string now